plugins/polio/api.py

```diff
@@ -1735,6 +1737,22 @@ def get_queryset(self):
         return Config.objects.filter(users=self.request.user)
 
 
+@swagger_auto_schema(tags=["datelogs"])
+class RoundDateHistoryEntryViewset(ModelViewSet):
+    http_method_names = ["get"]
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
+    serializer_class = RoundDateHistoryEntrySerializer
+    queryset = RoundDateHistoryEntry.objects.all()
```

Comment:
to respect proper permission this  should have a get_queryset method that limit the result to the campaign the user has access to